pulsar-client-cpp/docker-build.sh

```diff
@@ -20,6 +20,8 @@
 
 # Build Pulsar C++ client within a Docker container
 
+ARGS=$*
```

Comment:
Better to use an env variable (CMAKE_ARGS) in case someone wants to extend/modify the script.